rllib/utils/typing.py

```diff
@@ -104,9 +105,17 @@
 # Type of the config.policies dict for multi-agent training.
 MultiAgentPolicyConfigDict = Dict[PolicyID, "PolicySpec"]
 
+# A new stack Episode type: Either single-agent or multi-agent.
+EpisodeType = Union["SingleAgentEpisode", "MultiAgentEpisode"]
```

Comment:
This cannot be checked against during runtime. Does `[SingleAgentEpsiode", "MultiAgentEpisode"]` give a circular import?

Reply:
Yup, it does. I agree, this is annoying. See our offline discussion.